ee-feature-pack/galleon-common/src/main/resources/feature_groups/micrometer.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2019 Red Hat, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<!-- TODO Rename this file and the occurrences of 'micrometer' to what you set in TemplateExtension.SUBSYSTEM_NAME -->
```

Comment:
This line is totally bogus. :-). I imagine it's been copied from file to file dating back to an earlier provisioning mechanism that had nothing to do with Galleon. Like junk DNA that describes how to grow a tail.